pkg/cmd/pkg/package_repo_add.go

```diff
@@ -13,7 +15,16 @@ func newCmdPackageRepoAdd(ctx api.Context) *cobra.Command {
 		Short: "Add a package repository to DC/OS",
 		Args:  cobra.ExactArgs(2),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return invokePythonCLI(ctx)
+			c, err := cosmos.NewClient(ctx, pluginutil.HTTPClient(""))
+			if err != nil {
+				return err
+			}
+
+			_, err = c.PackageAddRepo(args[0], args[1], index)
+			if err != nil {
+				return err
+			}
+			return nil
 		},
 	}
 	cmd.Flags().IntVar(&index, "index", 0, "Numerical position in the package repository list. Package repositories are searched in descending order.")
```

Comment:
Normally the default here is not to send the index, then Cosmos will add it at the last position. Passing `--index=0` should be done manually.

This means we need to make the difference between the zero value and a missing value in client-go.

Reply:
It is differentiated in client-go by having 
```
type CosmosPackageAddRepoV1Request struct {
	Name  string `json:"name"`
	Uri   string `json:"uri"`
	Index int32  `json:"index,omitempty"`
}
```
Will differentiate it when calling the command.